attackstyles/src/main/java/net/runelite/client/plugins/attackstyles/AttackStylesPlugin.java

```diff
@@ -179,7 +187,7 @@ void onVarbitChanged(VarbitChanged event)
 
 		if (attackStyleVarbit != currentAttackStyleVarbit || equippedWeaponTypeVarbit != currentEquippedWeaponTypeVarbit || castingModeVarbit != currentCastingModeVarbit)
 		{
-			boolean weaponSwitch = currentEquippedWeaponTypeVarbit != equippedWeaponTypeVarbit;
+			boolean weaponSwitch = currentEquippedWeaponTypeVarbit == equippedWeaponTypeVarbit;
```

Comment:
Think the point was to only add a message if equipped weapon changed? So this should be `!=`

Reply:
Hmm I had to retest because I have no idea why I changed this. Anyways != is working as intended, thanks for the heads up.